### ISSUE TITLE
N4JSC.jar injection problem (Jetty port not injected, cannot run tests)

### ISSUE THREAD

**Issue by yoosiba** [eclipse/n4js#975]
Currently it is not possible to run tests with N4JSC.jar :
```
42593 INFO  TesterEventBus     - Registered: org.eclipse.n4js.hlc.base.testing.LoggingTestListener@200437e8
42608 WARN  JettyManager       - Port is already in use: 9415. Trying to use next available port: 60559.
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
44682 INFO  JettyManager       - Jetty instance has successfully started on '60559'.
{ FetchError: request to http://localhost:-1/n4js/testing/sessions/3dd163af-704e-411f-be49-1ebe17faa9f9/start failed, reason: connect ECONNREFUSED 127.0.0.1:80
    at ClientRequest.<anonymous> (/Users/me/myproject/node_modules/node-fetch/index.js:133:11)
    at emitOne (events.js:116:13)
    at ClientRequest.emit (events.js:211:7)
    at Socket.socketErrorListener (_http_client.js:387:9)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at emitErrorNT (internal/streams/destroy.js:64:8)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
  name: 'FetchError',
  message: 'request to http://localhost:-1/n4js/testing/sessions/3dd163af-704e-411f-be49-1ebe17faa9f9/start failed, reason: connect ECONNREFUSED 127.0.0.1:80',
  type: 'system',
  errno: 'ECONNREFUSED',
  code: 'ECONNREFUSED' }
```

most important parts are
```
Jetty instance has successfully started on '60559'.
request to http://localhost:-1/
```

Which suggests that there is an issue with setting the port.
Value used suggests the place holder value: https://github.com/eclipse/n4js/blob/master/plugins/org.eclipse.n4js.tester/src/org/eclipse/n4js/tester/internal/DefaultTestTreeTransformer.java#L155
that should be replaced at runtime with injected value: https://github.com/eclipse/n4js/blob/master/plugins/org.eclipse.n4js.tester/src/org/eclipse/n4js/tester/internal/DefaultTestTreeTransformer.java#L51

Looks like problem with injection of testers.

**Comment by lbeurerkellner** [comment]
I will look into this.

**Comment by lbeurerkellner** [comment]
This has been fixed in #978

**Comment by yoosiba** [comment]
works. thx
